### DISCUSSION TITLE
Why is the forum warning constantly displayed on the japanese subforum?

### DISCUSSION THREAD

**Discussion by ghost** [ScratchAddons/ScratchAddons#7528] [Help] [answered]
![kép](https://github.com/ScratchAddons/ScratchAddons/assets/150537842/7fe46944-fbae-4927-b4ac-f0302571e727)
https://github.com/ScratchAddons/ScratchAddons/blob/0da7e1c97052f0c02f4f03ff054374163d247fe2/content-scripts/cs.js#L154

Asking since it isn't forced to appear on other _non-english_ subforums

**Comment by CST1229** [comment, marked as answer]
I heard the Japanese forum doubles as a suggestions/bugs/etc forum, and it's also the biggest language subforum.

**Reply by ghost** [reply]
Oh, makes sense then.

**Reply by WorldLanguages** [reply]
The author of that line of code is Japanese, so it's a bit biased. But I assume it's true that Japanese users report bugs in their own language forum, so it's worth reminding users about turning off Scratch Addons and following the extension policy.
